src/main/resources/db/migration/mysql/V1.14__organisation-reporting-permission.sql

```diff
@@ -0,0 +1,10 @@
+DROP TABLE IF EXISTS `civil_servant_organisation_reporting_permission`;
```

Comment:
DROP TABLE should not be included here as it may lead to unexpected results.

Reply:
will remove